src/js/burials/config/form.js

```diff
@@ -153,18 +155,29 @@ const formConfig = {
           title: 'Claimant Contact Information',
           path: 'claimant-contact-information',
           uiSchema: {
+            'ui:validation': validateMatch('claimantEmail', 'view:claimantConfirmationEmail'),
```

Comment:
I think we could add a test to make sure there's a validation error when these don't match. To keep us from forgetting to change the property names here later if we make a change.